app/uk/gov/nationalarchives/omega/editorial/models/EditSet.scala

```diff
@@ -41,11 +41,12 @@ case class EditSetRecord(
   endDateYear: String,
   legalStatus: String,
   placeOfDeposit: String,
-  note: String
+  note: String,
+  relatedMaterial: Seq[RelatedMaterial] = Seq.empty
 )
 
 object EditSetRecord {
-  implicit val editSetRecordReads = Json.reads[EditSetRecord]
+  implicit val editSetRecordReads = Json.using[Json.WithDefaultValues].reads[EditSetRecord]
```

Comment:
I believe we need this to handle those (json) records where the `relatedMaterial` is missing - but is that a valid assumption, as opposed to expecting an empty array? 

For example:

```
{
   "ccr":"COAL 80/80/12",
   "oci":"COAL.2022.V12RJW.P",
   "scopeAndContent":"Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths.",
   "coveringDates":"1960",
   "formerReferenceDepartment":"",
   "startDateDay":"1",
   "startDateMonth":"1",
   "startDateYear":"1960",
   "endDateDay":"31",
   "endDateMonth":"12",
   "endDateYear":"1960",
   "legalStatus":"ref.1",
   "placeOfDeposit":"1",
   "note":"",
   "relatedMaterial":[
      
   ]
}
```

Reply:
I can't think of a case where you would want to differentiate between an empty collection and a property that doesn't exist for this specific case -

In general I use default parameters where I can to make refactoring easier. But if there's a case where the two cases have different meaning then I will change it.

Reply:
I'm not sure what the API contract will be.

My personal preference is to include all of the fields.

But that's not my call.

Let's call this resolved.